src/routes/(console)/organization-[organization]/+page.ts

```diff
@@ -28,15 +28,42 @@ export const load: PageLoad = async ({ params, url, route, depends, parent }) =>
         search: search || undefined
     });
 
+    let allProjects: typeof projects.projects = [];
+    let fetchedCount = 0;
+    const total = projects.total;
+
+    while (fetchedCount < total) {
+        const next = await sdk.forConsole.projects.list({
+            queries: [
+                Query.offset(fetchedCount),
+                Query.equal('teamId', params.organization),
+                Query.limit(limit),
+                Query.orderDesc('')
+            ],
+            search: search || undefined
+        });
+        allProjects = allProjects.concat(next.projects);
+        fetchedCount += next.projects.length;
+        if (next.projects.length === 0) break;
+    }
+
+    const allActiveProjects = allProjects.filter((p) => p.status === 'active');
```

Comment:
also, if status !== archived, then it's active project, currently status can be null,  active or archived, where both null and active means active